src/adapters/tests/leasing-adapter/offers.test.ts

```diff
@@ -77,6 +77,51 @@ describe(leasingAdapter.getOffersForContact, () => {
     assert(result.ok)
     expect(result.data).toEqual([expect.objectContaining({ id: offer.id })])
   })
+
+  it('should return 404 on not found', async () => {
+    nock(config.tenantsLeasesService.url)
+      .get('/contacts/P174965/offers')
+      .reply(404, {
+        content: {
+          ok: false,
+          err: 'not-found',
+        },
+      })
+
+    const res = await leasingAdapter.getOffersForContact('P174965')
+
+    expect(res.ok).toBe(false)
+    if (!res.ok) expect(res.err).toBe('not-found')
```

Comment:
```suggestion
    expect(res).toEqual({ ok: false, err: 'not-found' })
```
Smidigare sätt att göra assertions på adapter result, så slipper man if osv.. :)